k8s/object.bzl

```diff
@@ -402,6 +402,17 @@ _k8s_object_delete = rule(
     implementation = _common_impl,
 )
 
+# See "attrs" parameter at https://docs.bazel.build/versions/master/skylark/lib/globals.html#parameters-26
+_implicit_attrs = ["visibility", "deprecation", "tags", "testonly", "features"]
```

Comment:
Would you be willing to use each of these in a test somewhere?

Reply:
Request here would basically be to update something in the repo to include a testonly = 1, visibility = //foo:__subpackages__, etc so that when we bazel build //... something will fail if we lose support for this